### ISSUE TITLE
Incompatibility with latest HW:RM version.

### ISSUE THREAD

**Issue by demonfiren** [Fallen-Angel/CFHodEd#1]
It would appear that the same End Of Stream exception that occurs when attempting to open HW:RM files in old CFHodEd versions now also occurs in this one (both prerelase and stable versions on the releases page).
This suggests that Gearbox changed something again.

Given that this is probably the only way to get at the models I would like to request a fix.

**Comment by Mephistofeles** [comment]
Yes, I'm aware of this issue. HOD format changed significantly in 2.0 patch, meshes and textures are now stored in additional section and somewhat scrambled. Actually, I'm working on it as I have some free time, but you don't need to extract models from the new files - there is a way to download old assets and as far as I know the models themselves didn't change:
https://www.reddit.com/r/homeworld/comments/518exd/howto_download_old_versions_of_the_game_to/

**Comment by Mephistofeles** [comment]
I have managed to decompress and decode much of the data, there are still errors to fix, however [as you can see](http://i.imgur.com/6f5um8o.png), main features are read correctly.

**Comment by demonfiren** [comment]
Thank you.
This should make some things much simpler for the fandom.

**Comment by BreenBB** [comment]
I hope you fix that CFHodEd can't open HOD files in 2.0 patch, sadly I has GOG version of game and can't get older version using DepotDownloader.

**Comment by pandorazero** [comment]
Mephistofeles - I wish you success in correcting errors CFHodEd.

**Comment by pandorazero** [comment]
Is there any progress in the modification of CFHodEd?
Perhaps you'll lay out what's there?
